instrumentation/ratpack-1.4/testing/src/main/java/io/opentelemetry/instrumentation/ratpack/client/AbstractRatpackHttpClientTest.java

```diff
@@ -156,5 +156,9 @@ protected void configure(HttpClientTestOptions options) {
     options.disableTestReusedRequest();
 
     options.enableTestReadTimeout();
+
+    if (!Boolean.getBoolean("testLatestDeps")) {
+      options.disableTestHttps();
+    }
```

Comment:
just confirming, https tests worked in 1.4.0 but not in 1.4.1?

Reply:
I believe this is probably failing because of the change in the netty version. I'll try to look at it later in the evening maybe there is an easy way to make it work again.

Reply:
Now https test should work again. It was either testing with ratpack 1.5 (initializes ssl slightly differently) or using nettyy 4.1.31 (idk what it does to make it work), I went with netty.